packages/suite/src/components/onboarding/OnboardingLayout.tsx

```diff
@@ -164,13 +174,24 @@ type OnboardingLayoutProps = {
 export const OnboardingLayout = ({ children }: OnboardingLayoutProps) => {
     const theme = useSelector(state => state.suite.settings.theme);
 
+    const isMac = isMacOs();
+    const isDesktopApp = isDesktop();
+
+    const allowedModal = useFilteredModal(
+        [MODAL.CONTEXT_USER],
+        ['advanced-coin-settings', 'disable-tor'],
+    );
+
     return (
-        <TrafficLightOffset>
-            <Wrapper>
-                <SuiteBanners isOnboarding />
+        <TrafficLightOffset isVisible={false}>
+            <ConnectionGlobalModal />
+            {allowedModal && <ReduxModal {...allowedModal} />}
```

Comment:
I know this is moved code, but an opportunity to make it more explicit :slightly_smiling_face:  
```suggestion
            {allowedModal !== null ? <ReduxModal {...allowedModal} /> : null}
```